sample-plugin/pom.xml

```
@@ -922,11 +922,19 @@
   </build>
   <profiles>
     <!-- Use .1 of most recent active line, and .3 (or later) of previous lines: https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#choosing-a-version -->
+    <profile>
+      <id>2.426.x</id>
+      <properties>
+        <bom>2.426.x</bom>
+        <!-- TODO: Revise to 2.426.1 when release is available -->
+        <jenkins.version>2.426.1-rc34269.a_58a_1ee3f6d5</jenkins.version>
```

```suggestion
        <jenkins.version>2.426</jenkins.version>
```
Incremental versions must not be used in releases of anything, even `bom`.

---

Agreed.  This will be replaced with 2.426 before this is merged.  I wanted to use this pull request to evaluate the incremental from the stable-2.426 branch.  I should have marked this pull request as draft while using it that way.  I've made that change now.